tests/nunit.runner.tests.uwp/MainPage.xaml

```diff
@@ -5,7 +5,7 @@
     xmlns:local="using:NUnit.Runner.Tests"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:forms="using:Xamarin.Forms.Platform.WinRT"
```

Comment:
Templates project will also need this change. Can we make an issue when this is merged?